lib/osf-components/addon/components/collection-submission-card/template.hbs

```diff
@@ -0,0 +1,48 @@
+<div local-class='page-container'>
+    <div local-class='icon-container'>
+        <FaIcon @icon='clock' @prefix='far' />
+    </div>
+    <div local-class='text-container'>
+        <div local-class='project-title'>
+            {{t 'osf-components.moderators.all.button'}}
+        </div>
+        <div local-class='moderation-details'>
+            {{t
+                'osf-components.moderators.all.moderationDetails'
+                status=@collectionSubmission.status
+                date=@date
+                moderator=@moderator
+            }}
+        </div>
```

Comment:
This should be handled by the `<ReviewActionList>` component, but that's being updated in #1693  right now, so this should work as a placeholder until that is done